recipes/pspincalc/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "PSpincalc" %}
+{% set version = "0.2.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b8b5f36e72f90ea7b494d63248117f62b0895a019a955ece13f52a22b65b09d2
+
+build:
+  number: 0
```

Can it be ``noarch``?
```suggestion
  number: 0
  noarch: python
```